src/components/FlowRunStateTypeCount.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="flow-run-state-type-count">
+    <span class="flow-run-state-type-count__badge">
+      <template v-for="state in states" :key="state">
```

Comment:
Since the red and orange of failed and cancelled is so close in color, they may be largely indistinguishable and may even appear like a mistake. We may take this out, but I do appreciate how clean it is in code. Think it's something we can revisit on a design pass once we've got the page laid out.

Reply:
agreed, we can revisit 👍